[2025-01-06T14:38:20.805+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-06T14:38:20.812+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: transfer.transfer_Users manual__2025-01-06T14:38:19.852338+00:00 [queued]>
[2025-01-06T14:38:20.815+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: transfer.transfer_Users manual__2025-01-06T14:38:19.852338+00:00 [queued]>
[2025-01-06T14:38:20.815+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 1
[2025-01-06T14:38:20.820+0000] {taskinstance.py:2889} INFO - Executing <Task(GenericTransfer): transfer_Users> on 2025-01-06 14:38:19.852338+00:00
[2025-01-06T14:38:20.822+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=56) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-06T14:38:20.823+0000] {standard_task_runner.py:72} INFO - Started process 57 to run task
[2025-01-06T14:38:20.823+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'transfer', 'transfer_Users', 'manual__2025-01-06T14:38:19.852338+00:00', '--job-id', '15', '--raw', '--subdir', 'DAGS_FOLDER/tranfer_dag.py', '--cfg-path', '/tmp/tmp3hnx8200']
[2025-01-06T14:38:20.824+0000] {standard_task_runner.py:105} INFO - Job 15: Subtask transfer_Users
[2025-01-06T14:38:20.840+0000] {task_command.py:467} INFO - Running <TaskInstance: transfer.transfer_Users manual__2025-01-06T14:38:19.852338+00:00 [running]> on host 78501d4f6bfb
[2025-01-06T14:38:20.866+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='transfer' AIRFLOW_CTX_TASK_ID='transfer_Users' AIRFLOW_CTX_EXECUTION_DATE='2025-01-06T14:38:19.852338+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-06T14:38:19.852338+00:00'
[2025-01-06T14:38:20.866+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-06T14:38:20.874+0000] {base.py:84} INFO - Retrieving connection 'postgres-source'
[2025-01-06T14:38:20.882+0000] {base.py:84} INFO - Retrieving connection 'mysql-target'
[2025-01-06T14:38:20.883+0000] {generic_transfer.py:79} INFO - Extracting data from postgres-source
[2025-01-06T14:38:20.883+0000] {generic_transfer.py:80} INFO - Executing: 
 SELECT * FROM Users
[2025-01-06T14:38:20.885+0000] {base.py:84} INFO - Retrieving connection 'postgres-source'
[2025-01-06T14:38:20.888+0000] {sql.py:553} INFO - Running statement: SELECT * FROM Users, parameters: None
[2025-01-06T14:38:20.889+0000] {sql.py:562} INFO - Rows affected: 2
[2025-01-06T14:38:20.889+0000] {generic_transfer.py:107} INFO - Inserting rows into mysql-target
[2025-01-06T14:38:20.892+0000] {base.py:84} INFO - Retrieving connection 'mysql-target'
[2025-01-06T14:38:20.899+0000] {sql.py:694} INFO - Done loading. Loaded a total of 2 rows into Users
[2025-01-06T14:38:20.902+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-06T14:38:20.903+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=transfer, task_id=transfer_Users, run_id=manual__2025-01-06T14:38:19.852338+00:00, execution_date=20250106T143819, start_date=20250106T143820, end_date=20250106T143820
[2025-01-06T14:38:20.917+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-06T14:38:20.923+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-06T14:38:20.923+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
